README.md

```diff
@@ -37,6 +37,7 @@ In this example, the `wiki` folder will be synced with the wiki repo everytime a
 | `username` | Y | The repo owner's name. Used for pulling and pushing |
 | `access_token` | Y | An access token to use when pushing to the Wiki repo, can be set using `${{ secrets.GITHUB_TOKEN }}` |
 | `wiki_folder` | N | The folder to sync to the Wiki. <br/> <i>Default: `wiki`</i> |
+| `ignore_safe_warnings` | N | Fixes `fatal: detected dubious ownership in repository at '/wiki'` <br/> <i>Default: `false`</i> |
```

Comment:
This could include a more concise elaboration of *why* the optionality is necessary? I can find no clear articles to ignore directory safety in this situation.